app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
   },
   NetworkController: {
-    networkId: true,
     networkStatus: true,
+    selectedNetworkClientId: false,
+    networkId: true,
     providerConfig: {
-      nickname: true,
-      ticker: true,
+      rpcUrl: false,
       type: true,
+      chainId: false,
+      ticker: true,
+      nickname: true,
+      rpcPrefs: false,
+      id: false,
+    },
+    networkConfigurations: false,
+    networksMetadata: {
+      EIPS: true,
+      status: true,
     },
   },
+  NftController: {
+    allNftContracts: false,
+    allNfts: false,
+    ignoredNfts: false,
+  },
   OnboardingController: {
     completedOnboarding: true,
     firstTimeFlowType: true,
     seedPhraseBackedUp: true,
   },
   PreferencesController: {
-    currentLocale: true,
     featureFlags: true,
-    forgottenPassword: true,
+    // double-check if contains PID?
     ipfsGateway: true,
-    preferences: true,
+    identities: false,
+    lostIdentities: false,
+    selectedAddress: false,
+    useTokenDetection: true,
+    useNftDetection: true,
+    openSeaEnabled: true,
+    isMultiAccountBalancesEnabled: true,
+    disabledRpcMethodPreferences: true,
+    showTestNetworks: true,
+    isIpfsGatewayEnabled: true,
     useBlockie: true,
     useNonceField: true,
     usePhishDetect: true,
+    dismissSeedBackUpReminder: true,
+    useMultiAccountBalanceChecker: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    advancedGasFee: true,
+    incomingTransactionsPreferences: true,
+    // double-check if contains PID?
+    knownMethodData: true,
+    currentLocale: true,
+    forgottenPassword: true,
+    preferences: {
+      autoLockTimeLimit: true,
+      showFiatInTestnets: true,
+      showTestNetworks: true,
+      useNativeCurrencyAsPrimaryCurrency: true,
+      hideZeroBalanceTokens: true,
+    },
+    useAddressBarEnsResolution: true,
+    infuraBlocked: true,
+    ledgerTransportType: true,
+    snapRegistryList: true,
+    transactionSecurityCheckEnabled: true,
+    theme: true,
+    isLineaMainnetReleased: true,
+
+    securityAlertsEnabled: false,
+    snapsAddSnapAccountModalDismissed: true,
   },
   SignatureController: {
+    unapprovedMsgs: false,
+    unapprovedPersonalMsgs: false,
+    unapprovedTypedMessages: false,
     unapprovedMsgCount: true,
     unapprovedPersonalMsgCount: true,
     unapprovedTypedMessagesCount: true,
   },
+  SwapsController: {
+    swapsState: {
+      // double-check if contains PID?
+      quotes: false,
+      quotesPollingLimitEnabled: true,
+      fetchParams: true,
+      // double-check if contains PID?
+      tokens: true,
```

Comment:
Hmm, not entirely sure if this would be PII or not (probably not, since it should be the full token list our API returns?) but it would make sense to exclude just to reduce the size of this snapshot. It can get pretty large.

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/db9e2566e13811c8ddb6c590cc4370f8e74163f9